lib/logstash/outputs/base.rb

```diff
@@ -112,7 +112,7 @@ def output?(event)
 
     if !@exclude_tags.empty? && event["tags"]
       if @exclude_tags.send(@exclude_method) {|tag| event["tags"].include?(tag)}
-        @logger.debug? and @logger.debug("outputs/#{self.class.name}: Dropping event because tags contains excluded tags: #{exclude_tags.inspect}", event)
+        @logger.debug? and @logger.debug("outputs/#{self.class.name}: Dropping event because tags contains excluded tags: #{@exclude_tags.inspect}", event)
```

Comment:
I can do that but I suppose this is an optimization to avoid the evaluation of the string if debugging is not enabled. This seems a pattern used at sevaral places in logstash. What do you think?